src/UI/Factory.php

```diff
@@ -717,4 +717,25 @@ public function legacy($content);
 	 */
 	public function table();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Alerts inform the user about the state of the system. Such as the successful completion, the need for further
+	 *     input  of an actual error or stopping users in their tracks in high-risk tasks.
+	 *   composition: >
+	 *     Alerts are embedded or popup messages comprised of text and optionally buttons.
+	 *   effect: >
+	 *     Alerts convey information and optionally provide interaction by using Buttons.
+	 *
+	 * rules:
+	 *   interaction:
+	 *     1: Alerts MAY contain shortcuts displayed as Buttons.
```

Comment:
`shortcuts` mean links to other pages, right? These should not be `Buttons` but `Links`.

Reply:
We now have an optional link list